### PR TITLE
Classification finetuning v2

### PR DESCRIPTION
**Pull request by alex-matton**
Update classification finetuning with the new sagemaker release by taking into account the new output format.
Also slightly simplify one line of the notebooks where users had to avoid adding a trailing slash otherwise the upload would not work.

These changes are backward-compatible.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def has_ext_modules(foo) -> bool:
 
 
 setuptools.setup(name='cohere-sagemaker',
-                 version='0.7.2',
+                 version='0.8',
```
Comment (CoderHam) [inline setup.py]: Should be 0.8.0 right? Please remember to publish to pypi

File: cohere_sagemaker/classification.py [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
+    @property
+    def prediction(self) -> Prediction:
+        if isinstance(self.classification, (list, int, str)):
+            return self.classification
+        return self.classification["prediction"]
+
+    @property
+    def confidence(self) -> List[float]:
+        if isinstance(self.classification, (list, int, str)):
+            raise ValueError("Confidence scores are not available for version < 2 of classification-finetuning")
```
Comment (CoderHam) [inline cohere_sagemaker/classification.py]: This is the internal name. Maybe use the external name of the marketplace package?

Comment (CoderHam) [inline cohere_sagemaker/classification.py]:
```suggestion
            raise ValueError("Confidence scores are not available for versions prior to 2.0 of Classification Finetuning")
```

**Review by CoderHam — CHANGES_REQUESTED** [review]

**Review by jpekmez — DISMISSED** [review]

File: setup.py (hunk 1 above)
Comment (alex-matton) [reply setup.py]: Will publish to pypi once merged 👍

**Review by CoderHam — APPROVED** [review]

**Pull request merged**